.github/workflows/orca-k8s-example-test.yml

```diff
@@ -0,0 +1,87 @@
+        -e RUNTIME_DRIVER_PORT=54321 \
+        -e RUNTIME_EXECUTOR_INSTANCES=4 \
+        -e RUNTIME_EXECUTOR_CORES=8 \
+        -e RUNTIME_EXECUTOR_MEMORY=10g \
+        -e RUNTIME_TOTAL_EXECUTOR_CORES=32 \
+        -e RUNTIME_DRIVER_CORES=4 \
+        -e RUNTIME_DRIVER_MEMORY=10g \
+        -v /disk1/nfsdata/default-nfsvolumeclaim-pvc-d77a2e75-905f-49c1-ad1d-4eaee67c0967:/bigdl2.0/data
+        $IMAGE bash
+    - name: TF1 Examples Test
+      shell: bash
+      run: |
+        docker exec -i $CONTAINER_NAME bash -c "/opt/spark/bin/spark-submit \
+        --master ${RUNTIME_SPARK_MASTER} \
+        --deploy-mode client \
+        --conf spark.driver.host=${RUNTIME_DRIVER_HOST} \
+        --conf spark.driver.port=54321 \
+        --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark \
+        --name basic_text_classification \
+        --conf spark.kubernetes.container.image=$CONTAINER_NAME \
+        --conf spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName=nfsvolumeclaim \
+        --conf spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path=/bigdl2.0/data \
+        --conf spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName=nfsvolumeclaim \
+        --conf spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path=/bigdl2.0/data \
+        --conf spark.kubernetes.driverEnv.http_proxy=http://child-prc.intel.com:913 \
+        --conf spark.kubernetes.driverEnv.https_proxy=http://child-prc.intel.com:913 \
+        --conf spark.kubernetes.executorEnv.http_proxy=http://child-prc.intel.com:913 \
+        --conf spark.kubernetes.executorEnv.https_proxy=http://child-prc.intel.com:913 \
+        --conf spark.kubernetes.container.image.pullPolicy=Always \
+        --conf spark.pyspark.driver.python=/usr/local/envs/pytf1/bin/python \
+        --conf spark.pyspark.python=/usr/local/envs/pytf1/bin/python \
+        --conf spark.executorEnv.PYTHONHOME=/usr/local/envs/pytf1 \
+        --conf spark.executor.instances=4 \
+        --executor-cores 16 \
+        --executor-memory 50g \
+        --total-executor-cores 64 \
+        --driver-cores 4 \
+        --driver-memory 50g \
+        --properties-file /opt/bigdl-2.3.0-SNAPSHOT/conf/spark-bigdl.conf \
+        --py-files /opt/bigdl-2.3.0-SNAPSHOT/python/bigdl-orca-spark_3.1.3-2.3.0-SNAPSHOT-python-api.zip,/opt/bigdl-2.3.0-SNAPSHOT/examples/orca/learn/tf/basic_text_classification/basic_text_classification.py  \
+        --conf spark.driver.extraJavaOptions=-Dderby.stream.error.file=/tmp \
+        --conf spark.sql.catalogImplementation='in-memory' \
+        --conf spark.driver.extraClassPath=local:///opt/bigdl-2.3.0-SNAPSHOT/jars/* \
+        --conf spark.executor.extraClassPath=local:///opt/bigdl-2.3.0-SNAPSHOT/jars/* \
+        local:///opt/bigdl-2.3.0-SNAPSHOT/examples/orca/learn/tf/basic_text_classification/basic_text_classification.py \
```

Comment:
I think let's test for ncf tutorial?

Reply:
Sure, this will be updated to NCF later.